src/app/mangoapp_proto.h

```diff
@@ -10,13 +10,14 @@ struct mangoapp_msg_v1 {
 
     uint32_t pid;
     uint64_t visible_frametime_ns;
-    uint8_t fsrUpscale;
-    uint8_t fsrSharpness;
+    uint8_t scaler_filter;
+    uint8_t scaler_sharpness;
```

Comment:
We don't want to rename fields even if they won't be used, add these to the end of the struct instead

Reply:
Yes they should still be set by gamescope with the old data for backward compatibility.
If we want to remove fields it would need to be done by creating a v2 of the struct